packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/react/download-sample.md

```diff
@@ -0,0 +1,31 @@
+### Create a React app (optional)
+
+If you don't have an existing React app, you can quickly create an app by using the [Create React App command](https://create-react-app.dev/docs/getting-started/):
+
+```js
+npx create-react-app okta-app
+cd okta-app
+```
+
+### Install dependencies
+
+Add the [latest version of the Okta Sign-In Widget](https://github.com/okta/okta-signin-widget/releases) library to your React app. You can install it by using `npm` from your root app directory:
+
+```bash
+npm install @okta/okta-signin-widget@latest
+```
+
+You also need the [latest version of the Okta Auth JS SDK](https://github.com/okta/okta-auth-js/releases) (`@okta/okta-auth-js`) for Widget dependency:
+
+```bash
+npm install @okta/okta-auth-js@latest
+```
```

Comment:
This installs `"@okta/okta-auth-js": "^6.1.0",`

As a developer that writes a lot of tutorials, I typically do Okta installs on one line. For example:

```
npm install @okta/okta-signin-widget @okta/okta-auth-js
```